### ISSUE TITLE
make dynamic tabs actually ARIA 1.1 practices compliant

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#28918]
A breaking change, but would like to squeeze this into v5 if we can: turn dynamic tabs into actual "Tabs with automatic activation" (see https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html and https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel)

- only the currently selected/active tab receives focus, all other tabs are not focusable (`tabindex="-1"` added dynamically to them)
- cursor keys cycle between tabs, setting both the `tabindex="0"` and `aria-selected="true"` to the newly active tab (and `tabindex="-1"` / `aria-selected="false"` on the now not selected one)
- mouse behavior can remain as now (though need to flip `tabindex` accordingly)
- document this in the docs, referencing specifically the "automatic activation" route (so authors aren't confused/don't wonder why using `SPACE` or whatever does nothing)

**Comment by patrickhlauke** [comment]
it may even be worth making tabs a separate page/item in the navigation (and calling it "Tabs" as that's what many authors will be looking for in vain)

**Comment by patrickhlauke** [comment]
also, would love to (assuming it's buried in the code somewhere) remove any support for dropdowns inside dynamic tab interfaces. these are an abomination, and they don't map to any sane ARIA pattern

**Comment by patrickhlauke** [comment]
if not deemed too much of a breaking change, could consider backporting to v4 if reasonably straightforward...

**Comment by seh-notts** [comment]
Dear @patrickhlauke thank you so much for fixing this. Were you able to backport to v4?
